packages/features/auth/lib/next-auth-options.ts

```diff
@@ -437,6 +438,7 @@ export const AUTH_OPTIONS: AuthOptions = {
           role: existingUser.role,
           impersonatedByUID: token.impersonatedByUID as number,
           belongsToActiveTeam: token?.belongsToActiveTeam as boolean,
+          organizationId: token?.organizationId,
```

Comment:
Making sure we have the orgId in the session to then rely on it for branding provider